docs/design/keyboard-shortcuts.md

```diff
@@ -110,18 +143,17 @@ If there isn't one already, create a JSON file at the path that you use in devel
     }
     ```
 
-## Create a mapping of functions to actions
+## Create a mapping of actions to their functions
 
-The last major step is to map your custom functions onto the action names.
 
 1. Be sure that the HTML file that the `<FunctionFile>` element in the manifest points to has a `<script>` tag that loads a custom JavaScript file.
```

Comment:
```suggestion
1. In your project, open the JavaScript file loaded by your main HTML page.
```

It seems weird to get into details of specifying the manifest mapping, and also a script tag isn't always used for the load. Hoping this is a little clearer.

BTW:  Wouldn't you just call these associate APIs from your Office.onReady()?